rush.json

```diff
@@ -121,7 +121,7 @@
    * LTS schedule: https://nodejs.org/en/about/releases/
    * LTS versions: https://nodejs.org/en/download/releases/
    */
-  "nodeSupportedVersionRange": ">=16.13.0 <17.0.0 || >=18.12.0 <19.0.0",
+  "nodeSupportedVersionRange": ">=18.12.0 <19.0.0 || >=20.9.0 <21.0.0",
```

Comment:
- [ ] document lts policy, source of node version in ci, source of node version locally, steps for bumping node

Reply:
no lol don't lmao